plaidml/bridge/openvino/src/plaidml_plugin/plaidml_op_convolution.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <utility>
+#include <vector>
+
+#include "plaidml/core/core.h"
+#include "plaidml/edsl/edsl.h"
+#include "plaidml/exec/exec.h"
+#include "plaidml/op/op.h"
+
+#include "plaidml_op_convolution.hpp"
+#include "plaidml_util.hpp"
+
+using namespace plaidml::edsl;
+using namespace plaidml::exec;
+
+void OpConvolution::LoadWeights(State* state) {
+  conv_layer_ = dynamic_cast<ConvolutionLayer*>(layer_.get());
+
+  const auto& weight = conv_layer_->_weights;
+  const auto& bias = conv_layer_->_biases;
+
+  size_t O = conv_layer_->outData.front()->getDims()[1];
+  size_t I = conv_layer_->insData.front().lock()->getDims()[1] / conv_layer_->_group;
```

Comment:
I'm slightly surprised that dividing by the number of groups percolates as high as this, but I may be confused and this could be correct. I don't want to think through that now, but we should make sure to write a good test case that will catch problems here and come back to this once we have testing to verify our ideas.